src/login.rs

```diff
@@ -94,6 +95,29 @@ pub fn get_session(pid: Option<pid_t>) -> Result<String> {
     Ok(ss.to_string())
 }
 
+/// Determines the seat identifier of the seat the session identified
+/// by the specified session identifier belongs to.
+///
+/// Note that not all sessions are attached to a seat, this call will fail for them.
+pub fn get_seat<T: Into<Vec<u8>>>(session: T) -> Result<String> {
```

Comment:
Could you use `CStringArgument` in place of `Into<Vec<u8>>` here? See https://github.com/jmesmon/rust-systemd/blob/master/src/daemon.rs#L65 for existing usage.

Allows folks to pass `CStrs` directly if they so desire.